zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterContext.java

```diff
@@ -51,7 +52,7 @@ public static void remove() {
     allContexts.remove(Thread.currentThread());
   }
 
-  public static ConcurrentHashMap<Thread, InterpreterContext> getAllContexts() {
+  public static ConcurrentMap<Thread, InterpreterContext> getAllContexts() {
```

Comment:
This looks like a public API and is likely invoked by interpreter authors, I would not touch it.

Reply:
Okay, I have removed the change.